### ISSUE TITLE
[Article Proposal] - How to code a Pix2Pix GAN that brings out the artist in you

### ISSUE THREAD

**Issue by 01pooja10** [kili-technology/kili-blogger-program#43]
### My resource
In this blog, we will review the working of Pix2Pix Generative Adversarial Network and code/train it from scratch using the PyTorch framework. It will be trained on the edges2shoes dataset for generating realistic shoe images using user-drawn sketches.
**Topic:**
How to code a Pix2Pix GAN that brings out the artist in you
**Outline:**
1. Introduction
2. Pix2Pix Generator
3. Pix2Pix Discriminator
4. Loss function used
5. Data preparation and overview
5. Code walkthrough
6. Training and saving images
7. Conclusion
### My content is

- [x] A Kili Tutorial / Guide / How to article
- [ ] An Article

**Comment by theodu** [comment]
Hi @01pooja10 
Thank you for your article proposal
I think that it is very useful content but it doesn't fit in Kili's scope. It is too model-oriented. We won't select this article.

Happy New Year
